sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Triggers/ServiceBusTriggerBindingStrategy.cs

```diff
@@ -130,8 +130,7 @@ internal static void AddBindingData(Dictionary<string, object> bindingData, Serv
                 deliveryCounts[i] = messages[i].DeliveryCount;
                 deadLetterSources[i] = messages[i].DeadLetterSource;
                 lockTokens[i] = messages[i].LockToken;
-                //this is temporary until the Service Bus SDK addresses the missing timezone issue in case DateTime.MaxValue, github.com/Azure/azure-sdk-for-net/issues/15343
-                expiresAtUtcs[i] = messages[i].ExpiresAt.DateTime.ToUniversalTime();
```

Comment:
This is not an issue in the new SDK as we use DateTimeOffset.MaxValue, which is in UTC.